wordpress-mobile/WordPress-iOS
# Blogging Prompts: add feature description view

Ref: #18176 

This adds a description view to the Blogging Prompts Feature Introduction.
- A bordered stub view for the prompt card, to be populated later.
- The description paragraph.
- The bordered note.

Because we don't have any official way of showing the feature introduction just yet, I have set it to show in `applicationDidBecomeActive` to facilitate testing (by minimizing and relaunching the app). I will leave it for now and change/remove it later. 

To test:
- Enable `bloggingPrompts` feature flag.
- Run the app.
- When the app launches, the Feature Introduction should appear. 
- Verify:
  - A bordered area appears at the top under the header.
  - The `We'll show you...` paragraph is under that.
  - The bordered note is last, with the `Note:` text in bold.

<kbd>![bp_fi_content](https://user-images.githubusercontent.com/1816888/161162334-6ace237f-467c-4153-b1e5-89f6e9612566.png)</kbd>

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
